### DISCUSSION TITLE
New features

### DISCUSSION THREAD

**Discussion by SahityaRoy** [himanshu007-creator/SFC-foundations#131] [Ideas]
if we add dark mode features then its look good.

**Comment by divyashukla23** [comment]
Sure go ahead and send a PR. @SahityaRoy

**Reply by himanshu007-creator** [reply]
@SahityaRoy , its added

**Reply by himanshu007-creator** [reply]
@SahityaRoy , I would suggest to create a page template. The index.html file have nav-bar and footer, that will be same for all the pages of website, so raise an issue and create a <code>template.html</cod>
